### PR TITLE
fix(lnurlpay): handle status 500 for callback url #1995

### LINKED ISSUE #1995: [BUG] No error message is shown when payment to a lightning address fails
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

I have a Lightning address for my account but am incorrectly connected to my node.
If I try to pay using the extension on a different account, the send button spins and then goes back to its original state. There is no error message.

### Screenshots [optional]

image
image

### Steps To Reproduce

1. Create an Alby account at getAlby.com
2. Create a lightning address
3. Enter incorrect node credentials on the wallet tab
4. Try to send a payment to the lightning address from a different account using the Alby extension

### Expected behavior

I would expect an error message to be shown to the user - maybe to say that the destination is unreachable?

### Alby information

- Payment using Alby version 1.24.0, connected using hosted Alby account

### Device information

_No response_

### Additional context

_No response_

### Are you working on this?

None

### PR DESCRIPTION
**Pull request by escapedcat**
### Describe the changes you have made in this PR

Handle status 500 for callback-url

### Link this PR to an issue [optional]

Fixes #1995

### Type of change

- `fix`: Bug fix (non-breaking change which fixes an issue)

### Screenshots of the changes [optional]

image

### How has this been tested?

Manually

### CODE REVIEW

File: src/app/screens/LNURLPay/index.tsx [hunk 1]
```diff
@@ -129,6 +129,11 @@ function LNURLPay() {
           }
         );
 
+        if (response.status === 500) {
```
Comment (bumi) [inline src/app/screens/LNURLPay/index.tsx:132]: how about 501? :)

Comment (escapedcat) [reply src/app/screens/LNURLPay/index.tsx:132]: Should I just use `>== 500`?

File: src/app/screens/LNURLPay/index.tsx [hunk 2]
```diff
@@ -129,6 +129,11 @@ function LNURLPay() {
           }
         );
 
+        if (response.status >= 500) {
+          toast.error("Payment aborted: Could not connect to server");
```
Comment (bumi) [inline src/app/screens/LNURLPay/index.tsx:133]:
```suggestion
          toast.error("Payment aborted: Recipient server error");
```

**Review by bumi — COMMENTED** [review]
maybe we indicate that it is an error on the recipient side - not that we could not connect.

**Pull request merged**